src/Vue/Resources/assets/src/render_controller.ts

```diff
@@ -28,7 +28,7 @@ export default class extends Controller {
 
         this._dispatchEvent('vue:connect', { componentName: this.componentValue, props: this.props });
 
-        const component: Component = window.resolveVueComponent(this.componentValue);
+        const component: Component = (window as any).resolveVueComponent(this.componentValue);
```

Comment:
IMO the best option here would be to create a new types file, which declare the function  `resolveVueComponent` on `window`, and then configure it in the `package.json`.

Reply:
@Kocal, alternatively, possible to declare the `resolveVueComponent` function as global in the function declaration file itself.